database/index.js

```diff
@@ -200,8 +200,8 @@ createEvent = (creatorId, trailId, eventTitle, eventDesc, eventTrail, eventDate,
 };
 
 
-// get all events around the location
-getAllEventsNearLocations = (trailIdList) => {
+
+module.exports.getAllEventsNearLocations = (trailIdList) => {
```

Comment:
for uniformity, this module.exports should be down a the bottom? 

Can you make a quick fix of this?